File: 自写/Test/TestPriorityOfProtocolAndProperty/TestPriorityOfProtocolAndProperty/ViewController.m
//
//  ViewController.m
//  TestPriorityOfProtocolAndProperty
//
//  Created by luozhijun on 15-5-7.
//  Copyright (c) 2015年 luozhijun. All rights reserved.
//

#import "ViewController.h"
#import "ZJPickerView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    ZJPickerViewRow *row = [[ZJPickerViewRow alloc] init];
    ZJPickerViewDataSource *dataSource = [[ZJPickerViewDataSource alloc] init];
    row.subComponent = dataSource;
    row.titleForRow = @"aaaa";
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 自写/Test/TestBridingHeader_useframeworks/test/ViewController.swift
//
//  ViewController.swift
//  test
//
//  Created by luozhijun on 2016/10/18.
//  Copyright © 2016年 RickLuo. All rights reserved.
//

import UIKit
import AFNetworking
import IQKeyboardManager

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let view = MyView()
        view.backgroundColor = UIColor.redColor()
        view.frame = CGRect(x: 100, y: 100, width: 100, height: 100)
        self.view.addSubview(view)
        
        let mgr = AFHTTPSessionManager()
        print(mgr)
        
        let keyboardMgr = IQKeyboardManager.sharedManager()
        print(keyboardMgr)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

